app/api/contests/get_qualification_state.go

```diff
@@ -128,25 +125,28 @@ func (srv *Service) getContestInfoAndQualificationStateFromRequest(r *http.Reque
 		return nil, service.ErrInvalidRequest(err)
 	}
 
-	groupID, err := service.ResolveURLQueryPathInt64Field(r, "group_id")
-	if err != nil {
-		return nil, service.ErrInvalidRequest(err)
+	groupID := user.GroupID
```

Comment:
Ah you cannot use your new function here because it is another package. 😕
Cannot it be moved to the common service package? This is not something really related to items.

Reply:
We perform different checks here in the 'contest' package, so we don't need that new function from the 'items' package here.